config.go

```diff
@@ -209,12 +211,18 @@ func (config *TGFConfig) getAwsSession(duration int64) (*session.Session, error)
 				profile = "default"
 			}
 		}
-		log.Warningf("Your AWS configuration is set to expire your session in %v (automatically extended to %v)",
-			duration,
-			time.Duration(maxDuration)*time.Second)
+		if askedForMfa {
+			log.Warningf("Your AWS configuration is set to expire your session in %v. This timeout could not be automatically extended due to the session's MFA",
+				duration)
+		} else {
+			log.Warningf("Your AWS configuration is set to expire your session in %v (automatically extended to %v)",
```

Comment:
Could you make this `will extend by %v`,
and then after the next line you add a log saying `session extended successfully`?

Reply:
Hey, I'm not done. How did you find this PR?

Reply:
lol I don't know I was added as a reviewer :P

Reply:
In my last discussion with @JPLachance, we even discussed to put the MFA mandatory for all environments. So, that fix will be more than welcome.